### PR TITLE
Date ranges for copyright

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -173,30 +173,3 @@
       defend, and hold each Contributor harmless for any liability
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
```
Comment (JaredCorduan) [inline LICENSE:177]: fwiw, I kept the `END OF TERMS AND CONDITIONS` in the ledger license (but did definitely delete the appendix).

Comment (newhoggy) [reply LICENSE:177]: I put the line back.  Thanks!

Comment (newhoggy) [reply LICENSE:177]: I put the line back.  Thanks!

**Review by JaredCorduan — APPROVED** [review]
LGTM!

**Review by Jimbo4350 — APPROVED** [review]
LGTM!

**Pull request merged**